src/resources/postcss/tickets-admin/settings/_tickets-commerce.pcss

```diff
@@ -521,10 +522,19 @@
 }
 
 .tec-tickets__admin-settings-tickets-commerce-gateways-item {
-	align-items: center;
-	display: flex;
 	margin-bottom: var(--tec-spacer-7);
 	padding-bottom: var(--tec-spacer-0);
+	text-align: center;
+
+	& > div {
```

Comment:
Not completely sure that I understand what you're trying to do here. I mean, I see that you're trying to target a particular element. But I'm not sure why aren't you using a class for this first div?

Reply:
It's not just for the first div. It's for all child divs. If we add the toggle back in, it will apply to that as well and any other things we might add to that row. It's to ensure that there is sufficient padding above and below each element because they all stack vertically on mobile.

Reply:
@codingmusician Yep, I meant first level when I said first div. Why don't we have this padding to the gateway item directly? I haven't run it, but my interpretation is that the gateway item would be the "row" and we're applying top/bottom padding to the first childs (level). Maybe I'm missing something?